_episodes/14-modules.md

```diff
@@ -265,7 +279,9 @@ Command: seq       common transformation of FASTA/Q
          rename    rename sequence names
          randbase  choose a random base from hets
          cutN      cut sequence at long N
+         gap       get the gap locations
          listhet   extract the position of each het
+         hpc       homopolyer-compressed sequence
```

Comment:
These should be changes on a different branch to contribute upstream.

Reply:
Added to `mattgillucl-hpc-typos` branch from original repo.

Reply:
Removed.